api/appointmentsPOST.ts

```diff
@@ -0,0 +1,86 @@
+import handler from './handler';
+import db from './db';
+import { QueryResult } from 'pg';
+
+export default handler(async (request: any) => {
+
+  let connection = db();
```

Comment:
This needs to be updated to `withConnection` form; rebase onto development so that you get `withConnection` then fix this.